build.gradle

```diff
@@ -49,16 +50,33 @@ sourceSets {
   }
 }
 
+task preserveScripts(type: Copy) {
```

Comment:
Would `*Bin` be a better moniker than `*Scripts`? It does sit in the `bin` directory after all.

Reply:
Sure, I've updated the names accordingly.